pkgs/tools/misc/nixos-scripts/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, mpd_clientlib, curl, glib, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "0.1";
+  name = "nixos-scripts-${version}";
```

Comment:
This name suggests that these scripts are part of NixOS, which may be confusing.